### ISSUE TITLE
uncertainty_k_factor for logger_main_config table

### ISSUE THREAD

**Issue by kersting** [IEA-Task-43/digital_wra_data_standard#166]
The table logger_main_config has logger_acquisition_uncertainty but no uncertainty_k_factor. In order to get the logger_acquisition_uncertainty, correctly calculated, it is necessary to have also uncertainty_k_factor. 

Anyone opposed to the idea? I can also tackle this one.

**Comment by heikowestermann** [comment]
I agree. An alternative approach would be to define the k-factor and have it converted before entering the data model. I think it's best if this is done the same way in the whole model.

**Comment by stephenholleran** [comment]
Sounds like a good thing to do.

@heikowestermann do you mean specify that the k factor should be 2 for example and then the uncertainty would be converted to be for a k factor of 2?

**Comment by heikowestermann** [comment]
Correct. In my opinion, k=1 should be preferred as it is the value that is needed for further calculations. If nothing speaks against adding another key, it should be added as Gibson suggested, since there are already k_factor fields outside the dcc, if I remember correctly.

**Comment by stephenholleran** [comment]
Thanks @heikowestermann.
I wouldn't be keen on conversions so I would let the user type in the k value that is given to them from the documentation.
